contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -126,57 +126,72 @@ contract MultipleWinners is PeriodicPrizeStrategy {
     * @dev Set the prize split by passing an array MultipleWinnersPrizeSplit structs.
     * @param prizeStrategySplit Array of MultipleWinnersPrizeSplit structs.
   */
-  function setPrizeSplit(MultipleWinnersPrizeSplit[] memory prizeStrategySplit) external onlyOwner {
+  function setPrizeSplits(MultipleWinnersPrizeSplit[] memory prizeStrategySplit) external onlyOwner {
     uint256 _tempTotalPercentage;
 
-    for (uint256 index = 0; index < prizeStrategySplit.length; index++) {
+    for (uint8 index = 0; index < prizeStrategySplit.length; index++) {
         MultipleWinnersPrizeSplit memory split = prizeStrategySplit[index];
 
-        // If MultipleWinnersPrizeSplit is non-zero address set the prize split configuation.
-        if (split.target != address(0)) {
-            // Split percentage must be below 1000 and greater then 0 (e.x. 200 is equal to 20% percent)
-            // The range from 0 to 1000 is used for single decimal precision (e.x. 15 is 1.5%)
-            require(split.percentage > 0 && split.percentage <= 1000, "MultipleWinners/invalid-prizesplit-percentage-amount");
-            require(uint8(split.token) < 2, "MultipleWinners/invalid-ticket-type");
+        require(uint8(split.token) < 2, "MultipleWinners/invalid-prizesplit-token");
+        require(split.target != address(0), "MultipleWinners/invalid-prizesplit-target");
+        // Split percentage must be below 1000 and greater then 0 (e.x. 200 is equal to 20% percent)
+        // The range from 0 to 1000 is used for single decimal precision (e.x. 15 is 1.5%)
+        require(split.percentage > 0 && split.percentage <= 1000, "MultipleWinners/invalid-prizesplit-percentage");
```

Comment:
I think this validation should be moved below and look at the `_tempTotalPercentage`, to ensure the strat isn't over-allocating total splits.

Bonus: it'll only get called once.

Reply:
We could do this, but it would also allow for a `0` percentage to be passed in the configuration.

Is that worth giving up?